.github/workflows/linter.yml

```diff
@@ -0,0 +1,26 @@
+name: "Linter"
```

Comment:
I think we should use pre-commit here using the same settings we have in our own pre-commit. This way we guarantee that the code is checked in the same way